recipes/liburing/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+import platform
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LiburingConan(ConanFile):
+    name = "liburing"
+    license = "GPL-2.0-or-later"
+    homepage = "https://github.com/axboe/liburing"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    description = """helpers to setup and
+teardown io_uring instances, and also a simplified interface for
+applications that don't need (or want) to deal with the full kernel
+side implementation."""
+    topics = ("asynchronous-io", "async", "kernel")
+
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+
+    _autotools = None
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+
+        self._autotools = AutoToolsBuildEnvironment(self)
+        self._autotools.configure(configure_dir=self._source_subfolder)
+        return self._autotools
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{0}-{0}-{1}".format(self.name, self.version),
+                  self._source_subfolder)
+
+    def validate(self):
```

Comment:
```suggestion
    def configure(self):
```

Let's try it to check if we can see the error from CCI. Maybe, because validate runs after configure, CCI doesn't capture its output.

Reply:
Nothing new. Still fails.

Reply:
I gonna discuss with Conan team next week. It's first PR in CCI which requires Kernel version, and a new one.

Reply:
Thanks. I'll wait for news)

Reply:
@uilianries Any news?

Reply:
Yes, we had a meeting few hours ago we will need to check with infra team if it's possible to update kernel version or not. If they say no, we won't be able to accept this PR. Maybe Bincrafters can adopt it, because they are running Github Actions.